spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe CommentsController do
+  let(:user) { create(:learner) }
+  let(:rfc_with_comment) { create(:rfc_with_comment, user: user) }
+  let(:comment) { rfc_with_comment.comments.first }
+  let(:updated_comment) { comment.reload }
+  let(:perform_request) { proc { put :update, format: :json, params: {id: comment.id, comment: comment_params} } }
+
+  before do
+    allow(controller).to receive(:current_user).and_return(user)
+    perform_request.call
+  end
+
+  describe 'PUT #update' do
+    context 'with valid params' do
+      let(:comment_params) { {text: 'test100'} }
+
+      it 'saves the permited changes' do
```

Comment:
```suggestion
      it 'saves the permitted changes' do
```

🤓 Here and below.

Small improvement potential: Combine the scenarios (as the first one is included in the second). A single scenario (`it`) along `'applies the permitted changes'` should be sufficient.

Reply:
Thanks for correcting the wrong spelling.
I also changed the spec to have one instead of two scenarios.